### PR TITLE
Respect user permission in actions

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/lib/transitions/models.py [hunk 1]
```diff
@@ -346,9 +352,15 @@ def permission_info(self):
         }
 
     @classmethod
-    def transitions_for_model(cls, model):
+    def transitions_for_model(cls, model, user=None):
         content_type = ContentType.objects.get_for_model(model)
-        return cls.objects.filter(model__content_type=content_type)
+        transitions = cls.objects.filter(model__content_type=content_type)
+        if not user:
```
Comment (mkurek) [inline src/ralph/lib/transitions/models.py]: no need for this if (`_check_user_perm_for_transition` will handle it for you)

File: src/ralph/lib/transitions/models.py [hunk 2]
```diff
@@ -157,6 +157,15 @@ def _check_action_with_instances(instances, transition):
         validation_func(instances)
 
 
+def _check_user_perm_for_transition(user, transition):
+    if not user:
+        return True
```
Comment (szok) [inline src/ralph/lib/transitions/models.py:162]: Why true?

Comment (ar4s) [inline src/ralph/lib/transitions/models.py:162]: it's useful when execute from CLI whithout user

**Comment by mkurek** [comment]
:ok:

**Comment by szok** [comment]
:ok:

**Pull request merged**